### PR TITLE
#494 Favicon Added

### LINKED ISSUE #494: Add a favicon to the website
### Is your feature request related to a problem? Please describe.

I would like to add the icon to the website.

### Describe the solution you'd like

I would like to add the icon to the website.
image

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by PVBharadwaj**
## Related Issue

Closes #494 

## Description

The main index.html file is spotted out. An attempt to change the favicon is already done. But it was not reflected in the website as there is a div class before it. I removed it and the favicon is being displayed now.

## Screenshots
Before the change
image

After the change
image

## Checklist

- [x] My code adheres to the established style guidelines of the project.
- [] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,10 +3,6 @@
 
 <head>
 
-
-
-<div id="seo"></div>
```
Comment (AshAman999) [inline index.html:8]: this is important as well, can you trying moving seo after favicon

**Review by AshAman999 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (PVBharadwaj) [reply index.html:8]: sure

**Review by Viveksati5143 — APPROVED** [review]

**Comment by PVBharadwaj** [comment]
@AshAman999 I have made the required changes. Please review them.

**Review by AshAman999 — APPROVED** [review]

**Comment by PVBharadwaj** [comment]
@AshAman999  I am a GSSoC'23 contributor. The score for this PR is not added to the leaderboard. Please consider this PR under GSSoC'23

**Comment by PVBharadwaj** [comment]
> @AshAman999 I am a GSSoC'23 contributor. The score for this PR is not added to the leaderboard. Please consider this PR under GSSoC'23

@AshAman999  The score was still not added to the leaderboard. Please add GSSOC'23 label to the PR. It has been a month since the PR got merged.

**Comment by AshAman999** [comment]
Check now @PVBharadwaj

**Comment by PVBharadwaj** [comment]
@AshAman999  The score was not added. Please check.

**Pull request merged**